src/Analyzers/CSharp/Tests/NewLines/EmbeddedStatementPlacement/EmbeddedStatementPlacementTests.cs

```diff
@@ -339,5 +340,42 @@ void M()
                 Options = { { CSharpCodeStyleOptions.AllowEmbeddedStatementsOnSameLine, CodeStyleOption2.FalseWithSuggestionEnforcement } }
             }.RunAsync();
         }
+
+        [Fact, WorkItem(66017, "https://github.com/dotnet/roslyn/issues/66017")]
+        public async Task EmptyStatementAfterSwitch()
+        {
+            var source = @"
+class C
+{
+    void M(int i)
+    {
+        switch (i)
+        {
+            case 0:
+                break;
+        }[|;|]
+    }
+}";
+            var fixedCode = @"
+class C
+{
+    void M(int i)
+    {
+        switch (i)
+        {
+            case 0:
+                break;
+        }
+
+        ;
+    }
+}";
+            await new VerifyCS.Test
+            {
+                TestCode = source,
+                FixedCode = fixedCode,
+                Options = { { CSharpCodeStyleOptions.AllowEmbeddedStatementsOnSameLine, false } }
```

Comment:
❓ Why is this setting reporting a violation for the above code? I would expect this to be covered by the core formatting option "Leave statements and member declarations on the same line" instead.